src/app/App.tsx

```diff
@@ -98,6 +100,20 @@ export class App extends React.Component<Props> {
 
     Linking.addEventListener('url', this.handleOpenURL)
 
+    if (FIREBASE_ENABLED) {
+      this.dynamicLinksRemoveListener = dynamicLinks().onLink(({ url }) =>
+        this.handleOpenURL({ url })
+      )
+
+      if (Platform.OS === 'ios') {
```

Comment:
would be nice to add a comment about why we're not doing anything with android here (i'm curious for why as well :))

Reply:
There is a comment on line `79`, but I agree it would be helpful to clarify here too! I'll get it added.